backend/pokerogue/src/main/java/com/pokerogue/helper/external/s3/domain/FileExtension.java

```diff
@@ -0,0 +1,36 @@
+package com.pokerogue.helper.external.s3.domain;
+
+import com.pokerogue.helper.global.exception.ErrorMessage;
+import com.pokerogue.helper.global.exception.GlobalCustomException;
+import java.util.Arrays;
+import java.util.Optional;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum FileExtension {
+
+    JPG(".jpg"),
+    JPEG(".jpeg"),
+    PNG(".png"),
+    GIF(".gif"),
+    SVG(".svg");
+
+    private final String name;
+
+    public static String findExtensionName(String name) {
+        Optional<FileExtension> foundExtension = Arrays.stream(values())
+                .filter(fileExtension -> isSameFileExtension(name, fileExtension))
+                .findAny();
+
+        if (foundExtension.isEmpty()) {
+            throw new GlobalCustomException(ErrorMessage.FILE_EXTENSION_NOT_APPLY);
+        }
+        return foundExtension.get().getName();
```

Comment:
```suggestion
        if (foundExtension.isEmpty()) {
            throw new GlobalCustomException(ErrorMessage.FILE_EXTENSION_NOT_APPLY);
        }
        
        return foundExtension.get().getName();
```
`개행은 return 문 전에 반드시 한칸 띄운다.`

Reply:
`또한 이모지를 하나 이상 포함하도록 한다. `